python/source/org/libreoffice/ide/eclipse/python/export/AntScriptExportWizard.java

```diff
@@ -0,0 +1,135 @@
+/*************************************************************************
+ *
+ *$RCSfile: BuildScriptExportWizard.java,v $
+ *
+ * $Revision: 1.0 $
+ *
+ * last change: $Author: shobhanmandal $ $Date: 2018/03/03 18:36:29 $
+ *
+ * The Contents of this file are made available subject to the terms of
+ * the GNU Lesser General Public License Version 2.1
+ *
+ * GNU Lesser General Public License Version 2.1
+ * =============================================
+ * Copyright 2009 by Cédric Bosdonnat
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307 USA
+ *
+ * The Initial Developer of the Original Code is: Cédric Bosdonnat.
+ *
+ * Copyright: 2009 by Cédric Bosdonnat
+ *
+ * All Rights Reserved.
+ *
+ ************************************************************************/
+package org.libreoffice.ide.eclipse.python.export;
+
+import java.io.File;
+import java.util.Iterator;
+
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.IAdaptable;
+import org.eclipse.jface.dialogs.IDialogSettings;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.wizard.Wizard;
+import org.eclipse.ui.IExportWizard;
+import org.eclipse.ui.IWorkbench;
+import org.libreoffice.ide.eclipse.core.OOEclipsePlugin;
+import org.libreoffice.ide.eclipse.core.PluginLogger;
+import org.libreoffice.ide.eclipse.core.model.IUnoidlProject;
+import org.libreoffice.ide.eclipse.core.model.ProjectsManager;
+import org.libreoffice.ide.eclipse.python.Messages;
+import org.libreoffice.plugin.core.model.UnoPackage;
+
+/**
+ * Class for the new Ant Script Generation wizard.
+ */
+public class AntScriptExportWizard extends Wizard implements IExportWizard {
```

Comment:
We also don't need the Ant export here.

Reply:
Removed both the packages org.libreoffice.ide.eclipse.python.export & org.libreoffice.ide.eclipse.python.Test 

org.libreoffice.ide.eclipse.python.Test  : Unit test for the Manifest export page part UI controller.